src/test/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalkerIntegrationTest.java

```diff
@@ -13,17 +14,30 @@
     private static final String TEST_OUTPUT_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/tools/examples/";
 
     @Test
-    public void testExampleVariantWalker() throws IOException {
-        IntegrationTestSpec testSpec = new IntegrationTestSpec(
+    public void testExampleVariantWalkerNoSeqDict() throws IOException {
+        final IntegrationTestSpec testSpec = new IntegrationTestSpec(
                 " -L 1:100-200" +
                 " -R " + hg19MiniReference +
                 " -I " + TEST_DATA_DIRECTORY + "reads_data_source_test1.bam" +
                 " -V " + TEST_DATA_DIRECTORY + "example_variants.vcf" +
                 " -auxiliaryVariants " + TEST_DATA_DIRECTORY + "feature_data_source_test.vcf" +
                 " -O %s",
+                1,
+                UserException.MalformedGenomeLoc.class);
```

Comment:
I think we probably want `VariantWalker` to prefer the sequence dictionary from the driving variants only if there's an actual sequence dictionary (as opposed to an artificial one constructed from the index). I'll file a bug report.